src/components/ProductCont/ProductCard.jsx

```diff
@@ -0,0 +1,75 @@
+import axios from "axios";
+import {  useEffect } from "react";
+import { useProductContext } from "../context/ProductContext";
+import '../ProductCont/Product_page.css'
+
```

Comment:
suggestion - you may reduce the empty line spaces. this will reduce your lines of code.